README.md

```diff
@@ -1200,6 +1202,25 @@ liunx_pv_image
 uart:~$
 ```
 
+## Features and Tasks
+
+The following list shows the list of features already implemented and plans for the future release:
+
+* release v0.2:
```

Comment:
It is unclear if it is plan or already implemented. Could you please make it more clear?

Like "Included in release v0.2"

Also, could you please rewrite this and next paragraph to use the same tense and style in all the bullets?

Reply:
reworded.